packages/kbn-monaco/src/painless/antlr/painless_lexer.ts

```diff
@@ -184,13 +185,9 @@ export class painless_lexer extends Lexer {
 	// @Override
 	public sempred(_localctx: RuleContext, ruleIndex: number, predIndex: number): boolean {
 		switch (ruleIndex) {
-    // DO NOT CHANGE
-    // This is a manual fix to handle slashes appropriately
 		case 31:
 			return this.DIV_sempred(_localctx, predIndex);
 
-    // DO NOT CHANGE
-    // This is a manual fix to handle regexes appropriately
 		case 77:
```

Comment:
Similarly to above - I think we need to add this comment back in, and change it to `78` based on the addition of `DOLLAR`.

Reply:
Done.